README.md

```diff
@@ -0,0 +1,34 @@
+# Dart
+
+[Dart][website] is an open-source, scalable programming language, with robust
+libraries and runtimes, for building web, server, and mobile apps.
+
+## Using Dart
+
+Visit the [Dart website][website] to learn more about the
+[language][lang], [tools][tools],
+[tutorials][codelab], and more.
```

Comment:
tutorials -> getting started

(the link isn't to tutorials, and you wouldn't want to learn *about* tutorials anyway)